README.md

```diff
@@ -1,5 +1,52 @@
 # docassemble.assemblylinewizard
 
+## Developing Tests
+
+To write and run the tests, you'll need to set up your testing environment in Python.
+
+### Set up a virtual environment
+
+First, set up a virtual environment called `docassemble` to hold all the packages
+related to this repository.
+This will keep your code clean, and make sure this repository does not interfere with
+your other projects.
+
+```
+$ cd docassemble/assemblylinewizard
+$ pip3 install virtualenv
+$ virtualenv -p $(which python3) docassemble
+$ source docassemble/bin/activate
+$ pip install -r requirements.txt
+```
+
+(I used `pip3` and `python3` above, because I have both Python 2 and Python 3 on my machine, but you
+can use just `pip` and `python` if you only have Python 3.) Be forewarned, as part of the requirements,
+you have to install the `docassemble` package, which is complex. If you have trouble downloading that

```

Comment:
Maybe this should go at the very top. That is, if this is the hardest step and they can't get past that, better that they find that out before going through the bother of the other stuff.